### DISCUSSION TITLE
Entwickler

### DISCUSSION THREAD

**Discussion by meerfrau** [openmdx/openmdx#89] [General]
Guten Tag,

es gibt gerade eine Debatte in der Wikipedia, ob openMDX wirklich von [diesem Harald Burger](https://de.wikipedia.org/wiki/Harald_Burger) entwickelt wird.

Da dort jeder editieren kann könnte es sich durchaus um eine Falschzuschreibung handeln. Können Sie persönlich das vielleicht aufklären?

Mit freundlichen Grüßen!

**Comment by meerfrau** [comment]
@wfro64 Falls sie helfen können, natürlich auch gerne.

**Comment by DlRTY-HARRY** [comment]
Hallo @meerfrau 

Prof. Burger und ich wurden schon öfters verwechselt. Während er sich im Bereich der Sprachwissenschaften verdient gemacht hat, bewege ich mich eher in der Informatik. Davon abgesehen bin ich leichter auf [LinkedIn](https://www.linkedin.com/in/burger/) als auf Wikipedia zu finden.
